Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["darknet", "machine-learning", "deep-learning", "neural-networks", "
 readme = "./README.md"
 
 [dependencies]
-darknet-sys = "0.3.1"
+darknet-sys = { version = "0.3.2", default-features = false }
```

Comment:
This assumes a new patch version of `darknet-sys` will be released with the prerequisite change.